react/fields/MonthPicker/NativeMonthPicker/NativeMonthPicker.js

```diff
@@ -49,6 +49,7 @@ export default class NativeMonthPicker extends Component {
     super();
 
     this.handleChange = this.handleChange.bind(this);
+    this.handleBlur = this.handleBlur.bind(this);
```

Comment:
It's actually not for the prefill bug. The reason the onBlur was added is because the default onBlur implementation was passing back a string value rather than our custom object. This was playing havoc with our forms library so I added a new onBlur that ensures our custom value.